subjects/ascii-art-reverse.en.md

```diff
@@ -0,0 +1,42 @@
+## ascii-art-reverse
+
+### Objectives
+
+Ascii-art-reverse consists on reversing the process, convertin the graphic representation into a text.
+
+- You will have to create a `txt` file with a graphic representation of a random phrase;
```

Comment:
I don't see why `txt` should be highlighted like this, why not write "text file" instead of "`txt` file" ?